lib/screen/shopping_cart_screen.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import 'package:food_truck_mobile/widget/cart_item.dart';
+import 'package:food_truck_mobile/widget/section_divider.dart';
+import 'package:food_truck_mobile/widget/text.dart';
+
+import '../helper/constants.dart';
+
+class ShoppingCart extends StatelessWidget {
+  final List<List> cartItems;
+
+  // cartItems should be passed in, this is only for UI purposes
+  const ShoppingCart(
+      {Key? key,
+      this.cartItems = const [
+        ["Artichoke Sandwich", 7.50, 2],
+        ["Egg Sandwich", 5.70, 1]
+      ]})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: const Text("Shopping cart"),
+        ),
+        body: Padding(
+            padding:
+                const EdgeInsets.symmetric(vertical: 5.0, horizontal: 15.0),
+            child: ListView(
+              children: [
+                const TextTitleLarge(
+                  text: "Shopping cart",
+                  isBold: true,
+                ),
+                const SectionDivider(),
+                ..._getContent(),
+                const Row(
```

Comment:
Line 37-48 can be extracted as a new widget in widget/check_out_item.dart, it takes 2 attribute: name and price